share/zmtest

```diff
@@ -4,35 +4,36 @@ bindir="$(dirname "$0")"
 
 ZMB="${bindir}/zmb"
 JQ="$(which jq)"
+ECHO=/bin/echo
 
 usage () {
     status="$1"
     message="$2"
-    echo "${message}" >&2
-    echo "Usage: zmtest [OPTIONS] DOMAIN" >&2
-    echo >&2
-    echo "Options:" >&2
-    echo "  -s URL --server URL   Zonemaster Backend to query. Default is http://localhost:5000/" >&2
-    echo "  --noipv4              Run the test without ipv4." >&2
-    echo "  --noipv6              Run the test without ipv6." >&2
-    echo "  --lang LANG           A language tag. Default is \"en\"." >&2
-    echo "                        Valid values are determined by backend_config.ini." >&2
+    "${ECHO}" "${message}" >&2
+    "${ECHO}" "Usage: zmtest [OPTIONS] DOMAIN" >&2
+    "${ECHO}" >&2
+    "${ECHO}" "Options:" >&2
+    "${ECHO}" "  -s URL --server URL   Zonemaster Backend to query. Default is http://localhost:5000/" >&2
+    "${ECHO}" "  --noipv4              Run the test without ipv4." >&2
+    "${ECHO}" "  --noipv6              Run the test without ipv6." >&2
+    "${ECHO}" "  --lang LANG           A language tag. Default is \"en\"." >&2
+    "${ECHO}" "                        Valid values are determined by backend_config.ini." >&2
     exit "${status}"
 }
 
 error () {
     status="$1"
     message="$2"
-    echo "error: ${message}" >&2
+    "${ECHO}" "error: ${message}" >&2
     exit "${status}"
 }
 
 zmb () {
     server_url="$1"; shift
     json="$("${ZMB}" --server="${server_url}" "$@" 2>&1)" || error 1 "method $1: ${json}"
-    json="$(echo "${json}" | "${JQ}" -S . 2>/dev/null)" || error 1 "method $1 did not return valid JSON output"
-    error="$(echo "${json}" | "${JQ}" -e .error 2>/dev/null)" && error 1 "method $1: ${error}"
-    echo "${json}"
+    json="$("${ECHO}" -E "${json}" | "${JQ}" -S . 2>/dev/null)" || error 1 "method $1 did not return valid JSON output"
+    error="$("${ECHO}" -E "${json}" | "${JQ}" -e .error 2>/dev/null)" && error 1 "method $1: ${error}"
+    "${ECHO}" -E "${json}"
```

Comment:
`-E` is not supported as an option by `/bin/echo` in FreeBSD. "\n" is nothing special either. 
```
$ /bin/echo -E text
-E text
$ /bin/echo "text\n"
text\n
```

Reply:
I replaced /bin/echo with printf.